packages/playground/src/layouts/SandboxPullRequestCompareLayout.tsx

```diff
@@ -85,6 +86,15 @@ const SandboxPullRequestCompare: React.FC<SandboxPullRequestCompareProps> = ({
     }
   })
 
+  useEffect(() => {
+    if (commitData && commitData.length > 0) {
+      reset({
+        title: commitData[commitData.length - 1].title,
```

Comment:
Is `title` optional? if yes, lets use optional chaining `?.title`

Reply:
fixed